src/configure/resources/tracePoints.ts

```diff
@@ -24,5 +24,9 @@ export class TracePoints {
     public static InitializeDynamicValidation = 'InitializeDynamicValidation';
     public static SetInputControlValueFromRepoAnalysisResult = 'setInputControlValueFromRepoAnalysisResult';
     public static TemplateServiceCallFailed = 'TemplateServiceCallFailed';
-    public static GetTemplateFile = 'GetTemplateFile'
+    public static GetTemplateFile = 'GetTemplateFile';
```

Comment:
It would be good if you can write a comment for each trace point - telling what kind of action errors will this trace point capture. This will help us in writing kusto queries when trying to figure our some specific falures.
Do it only for the new trace points. That should be okay.

Reply:
Updated the TracePoints for more clarity